web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -96,6 +110,7 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
       }
       setGroupList(result);
     } else {
+      updateURL(e.target.value);
```

Comment:
If we do the same URL update in both the `if` + `else` branches, would it be better to put it outside of the conditional, like right at the start of the function?

Reply:
ready! ;)